index.bs

```diff
@@ -261,15 +265,15 @@ interface Sanitizer {
 };
 </pre>
 
-Note: {{Sanitizer}} will likely get an additional method:
-    <br>`[NewObject] static Sanitizer getDefault();`
-
 A {{Sanitizer}} has an associated <dfn for="Sanitizer">configuration</dfn>, a {{SanitizerConfig}}.
 
 <div algorithm>
 The <dfn for="Sanitizer" export>constructor</dfn>(|configuration|)
 method steps are:
 
+1. If |configuration| is a {{SanitizerPresets}} [=string=], then:
+    1. [=Assert=]: |configuration| [=is=] {{SanitizerPresets/default}}.
+    1. Let |configuration| be the [=built-in safe default configuration=].
```

Comment:
```suggestion
    1. Set |configuration| to the [=built-in safe default configuration=].
```

Reply:
I'm using "Let |...| be" quite a lot. Not sure where I had picked this up. Is that generally undesired; or only here (where I'm updating a variable, rather than initializing it)?